### PR TITLE
New package: nim-0.10.2

### PR DESCRIPTION
**Pull request by allan**

### CODE REVIEW

File: srcpkgs/nim/template [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Template file for 'nim'
+pkgname=nim
+version=0.10.2
+revision=1
+depends="libzip gcc gcc-c++"
```
Comment (xtraeme) [inline srcpkgs/nim/template]: `gcc` is a dependency of `gcc-c++`, you can remove it.

File: srcpkgs/nim/template [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+makedepends="libzip"
```
Comment (xtraeme) [inline srcpkgs/nim/template]: Probably you do need to add `libzip-devel` to `makedepends` and remove `libzip` from `depends`.

File: srcpkgs/nim/template [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+hostmakedepends="git ed"
+short_desc="Nim language"
```
Comment (xtraeme) [inline srcpkgs/nim/template]: Any chance to improve the short description somehow?

File: srcpkgs/nim/template [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+post_extract() {
+	cd $wrksrc
+	test -d csources ||
+	git clone --depth=1 https://github.com/nim-lang/csources
+}
+
+do_build() {
+	if [ "$CROSS_BUILD" ]; then
+		ed >/dev/null 2>&1 config/nim.cfg <<-EDIT
```
Comment (xtraeme) [inline srcpkgs/nim/template]: Assuming cross compilation will always be for arm is completely wrong. Use something like this:

```
case "$XBPS_TARGET_MACHINE" in
 arm*) ...
esac
```

File: srcpkgs/nim/template (hunk 2 above)
Comment (allan) [inline srcpkgs/nim/template]: It doesn't need headers from libzip-devel.  Only used for the installer it compiles for itself. Maybe it should go in hostmakedepends then?

**Pull request merged**